backend/src/main/java/com/bakdata/conquery/apiv1/MeProcessor.java

```diff
@@ -49,8 +78,17 @@ public FEMeInformation getUserInformation(@NonNull User user){
 	public static class FEMeInformation {
 		String userName;
 		boolean hideLogoutButton;
-		List<FEPermission> permissions;
+		Map<DatasetId, FEDatasetAbility> datasetAbilities;
 		List<IdLabel<GroupId>> groups;
 	}
 
+	/**
+	 * Front end container to describe what the user can do on a
+	 * dataset by simple flags.
+	 */
+	@Builder
+	private static class FEDatasetAbility {
+		private final boolean canUpload;
```

Comment:
Das wird ja vermutlich auch in der Autodoc auftauchen, kannst du hier Doku mit einem Querverweis auf die entsprechende Ability machen?

Reply:
Ja, guter Punkt